src/rbac-forms/tests/rbacUserTagsList.test.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import toJson from 'enzyme-to-json';
+import { RbacUserTagsList } from '../';
+
+describe('Rbac user tags list component', () => {
+  let tags;
+  let tenant;
+
+  beforeEach(() => {
+    tags = [{
```

Comment:
You can import data from stories so we don't have to change data set on 2 places whenever API changes.